scripts/s3-utils.js

```diff
@@ -3,7 +3,7 @@
 const path = require('path')
 const s3 = require('s3-client')
 const { s3Prefix, s3Bucket } = require('../src/server/config')
-const { remove, move, ensureDir } = require('fs-extra')
```

Comment:
same here - why don't we detect this automatically?

Reply:
I don't know, I thought we did and that's why these are still here. It's only after I started using the TS linter more that I noticed.